test/service/test_response.py

```diff
@@ -4052,3 +4060,404 @@ def test_projects_response(self):
             'accessible': True
         }
         self.assertEqual(expected_project, project)
+
+    def test_tdr_project_field_sorting(self):
+        """
+        Verify the response when sorting by project fields
+        """
+        test_data = {
```

Comment:
Surely, there is a better way to assert the ordering of the response without a hard-coded literal, especially considering that it would be extremely tedious to maintain this list, and considering that it is somewhat opaque as to why these literal list items are ordered the way they are.